<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CALCULATOR</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="AC">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(245, 121, 0);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>CLEAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DEL">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(245, 121, 0);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>DEL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EXIT">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(245, 121, 0);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="push4">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="push8">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="push7">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="push5">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="push6">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="push1">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="push9">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushmin">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(117, 80, 123);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushadd">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(117, 80, 123);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="push2">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="push3">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushx">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(117, 80, 123);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushdot">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="push0">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushequal">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(78, 154, 6);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushdiv">
        <property name="styleSheet">
         <string notr="true"> width: 30px;
 height: 30px;
background-color: rgb(117, 80, 123);
border: 1px solid rgb(233, 185, 110);
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
